src/Microsoft.ML.StandardTrainers/Standard/Online/OnlineGradientDescent.cs

```diff
@@ -25,7 +25,7 @@ namespace Microsoft.ML.Trainers
 {
     /// <summary>
     /// The <see cref="IEstimator{TTransformer}"/> for training a linear regression model using
-    /// Online Gradient Descent (OGD) for estimating the parameters of the linear regression model.
+    /// <a href="https://en.wikipedia.org/wiki/Gradient_descent">Online Gradient Descent (OGD)</a> for estimating the parameters of the linear regression model.
```

Comment:
Same comment here. But again, it might not be an issue.

Reply:
don't think we have issues with `<a href>` inside xml. 
Probably markdown